certora/specs/CER-2-Operator/CER-68-Operator-authorization.spec

```diff
@@ -0,0 +1,119 @@
+import "../utils/IsMustRevertFunction.spec";
```

Comment:
this file only verifies that only owner can call `setOperator`. however, it should also verify that only owner can call `setAccountOperator`. the requirement pertains to both functions (all the methods that set the operator)

Reply:
Hi Kasper. To Address your request, I added `onlyOwnerOrOperatorCanCallSetAccountOperator`. See if it checks what you are hoping for.